src/bionics.cpp

```diff
@@ -85,20 +203,26 @@ bionic_data const& bionic_info(std::string const &id)
 
     debugmsg("bad bionic id");
 
-    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0, "bad_bionic", false};
+    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0,
+                                         "bad_bionic", false,
+                                         { num_bp , 1 } };
```

Comment:
Failure of what kind? Jenkins didn't detect anything. GCC compliation error?

Try `std::make_pair( num_bp, 1 )`.